srcpkgs/NetworkManager-strongswan/template

```diff
@@ -13,3 +13,4 @@ license="GPL-2.0-or-later"
 homepage="https://www.strongswan.org/"
 distfiles="https://download.strongswan.org/NetworkManager/NetworkManager-strongswan-${version}.tar.bz2"
 checksum=343b68cbe32f65e2baa01b37716415e4360addc8c90209d45504b52b8864bd04
+configure_args="--without-libnm-glib --disable-more-warnings"
```

Comment:
This will override `configure_args` set at line #6

Reply:
fixed, thanks!